04LessonFour/in-class-project/index.html

```diff
@@ -1,10 +1,83 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>The Daily Planet</title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
-
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Intel</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <header>
+
+        <h1>Intel</h1>
+
+        <nav>
+
+            <a href="#">i7 Processor</a>
+
+            <a href="#">Solid-State Drives</a>
+
+        </nav>
+
+    </header>
+
+    <div class="paragraphs">
+
+        <h3>Intel&reg Core&trade; i7</h3>
+
+        <p>The 6th generation Intel&reg Core&trade; processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lightning fast speeds and peak performance through even
+            the toughest of tasks and games.</p>
+
+    </div>
+
+    <div class="first_paragraph">
+
+        <h3>Ready. Wake. Create.</h3>
+
+        <p>
+            From zero to power in a flash, the 6th Generations Intel&reg Core&trade; processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever.
+        </p>
+
+    </div>
+
+    <div class="picture_1">
+
+        <img src="img/processor.jpg" />
+
+    </div>
+
+    <div id="support">
+
+        <h3>Intel Customer Support</h3>
+
+        <p>
+            Find Support Topics by Product, Drivers and Software, or ask the community <a href="#">read more.</a>
+        </p>
+
```

Comment:
great work classing and iding your divs! consider using mains and sections to make it a little less confusing later on if you strictly use divs.